clients/cobol-lsp-vscode-extension/src/services/BridgeForGitLoader.ts

```diff
@@ -55,13 +60,18 @@ export function decodeBridgeJson(json: unknown): B4GTypeMetadata | undefined {
 
 export async function loadBridgeJsonContent(
   documentUri: Uri,
-): Promise<unknown> {
+): Promise<B4GTypeMetadata | undefined> {
+  const docUriString = documentUri.toString();
+  if (docToBridgeJsonContentMap.has(docUriString)) {
```

Comment:
Wouldn't it be sufficient to just use the `b4gPath` below as the key and drop `bridgeJsonToDocUriMap`?

Reply:
updated